materializer/nestedfieldselection/README.md

```diff
@@ -0,0 +1,132 @@
+`@materializer` is a great way to seamlessly extend a type with data from another type by executing an operation, sometimes with arguments supplied by the extended type. 
+
+Sometimes the data needed is nested, so @materializer provides the ability to 
```

Comment:
Doesn't really explain what is the purpose here. 

_Sometimes for schema curation or cleanliness the desired value of the field is from a nested value of the operation, for example just picking a `city` from an `Address` type._